### PR TITLE
fix: handle bar chart flickering

### PR DESCRIPTION
**Pull request by pdp2121**
## High Level Overview of Change

The bar chart for Upgrade Status is flickering due to:

1. Bar animation per data refetch
2. A strange data point from validators list with undefined fields:
`{"ledger_index":74661353,"ledger_hash":"613E298A8C0AEB816D16AA61952E0834BBD9B5E5677EA3E9A2413118EE074363"}`

This PR will fix the above issues.

### Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### TypeScript/Hooks Update

- [ ] Updated files to React Hooks
- [ ] Updated files to TypeScript

### CODE REVIEW

**Review by JST5000 — APPROVED** [review]
LGTM

File: src/containers/Network/UpgradeStatus.tsx [hunk 1]
```diff
@@ -26,14 +26,17 @@ export const UpgradeStatus = () => {
     const tempData: any[] = []
     validators.reduce((aggregate, current) => {
       const aggregation = { ...aggregate }
-      if (!aggregation[current.server_version]) {
-        aggregation[current.server_version] = {
-          server_version: current.server_version,
+      const currentVersion = current.server_version
```
Comment (ckniffen) [inline src/containers/Network/UpgradeStatus.tsx:29]: would `current.server_version ?? null` work?

**Comment by ckniffen** [comment]
Can you add a test for this weird case.

File: src/containers/Network/UpgradeStatus.tsx [hunk 2]
```diff
@@ -111,6 +114,11 @@ export const UpgradeStatus = () => {
   }
 
   const validatorCount = Object.keys(vList).length
+  Object.values(vList).forEach((v: any) => {
+    if (v.signing_key === undefined) {
+      console.log(v)
```
Comment (jonathanlei) [inline src/containers/Network/UpgradeStatus.tsx:119]: What's the console.log for?

Comment (pdp2121) [reply src/containers/Network/UpgradeStatus.tsx]: Removed it.

**Review by jonathanlei — APPROVED** [review]
LGTM now

**Comment by mvadari** [comment]
Do we know what's causing that weird test case? Is there a bug in the VHS?

**Comment by pdp2121** [comment]
> Do we know what's causing that weird test case? Is there a bug in the VHS?

It's the Streams component of the Explorer that caused this issue as we investigated. Will resolve it in another ticket.

File: src/containers/Network/test/upgradeStatus.test.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        incomplete: false,
+      },
+      chain: 'chain.4',
+      partial: false,
+    },
+  ]
+
+  it('aggregarteData handle edge case', () => {
+    expect(aggregateData(undefinedData)).toEqual([
+      { count: 1, label: ' N/A ', value: 50 },
```
Comment (ckniffen) [inline src/containers/Network/test/upgradeStatus.test.js:42]: I think N/A should not have any matches.  Any validator without a pubkey isn't a valid validator.

Comment (pdp2121) [reply src/containers/Network/test/upgradeStatus.test.js:42]: Okay I will not count ones with undefined pubkey toward calculation then.

**Review by ckniffen — APPROVED** [review]
